easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.5.1-GCC-12.2.0.eb

```
@@ -0,0 +1,55 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp
+# Modified by: Sebastien Moretti for non-x86_64 systems
+# SIB Swiss Institute of Bioinformatics
+# Modified by: Bente Barge <bente.s.barge@uit.no>
+
+name = 'Bowtie2'
+version = '2.5.1'
+_simde_version = '20220504'
+_simde_commit = 'cbef1c1'
```

since we're not using a release version, perhaps we should bump to the latest commit?
```suggestion
_simde_version = '20230122'
_simde_commit = '6e9ac24'
```